icd/api/app_shader_optimizer.cpp

```diff
@@ -143,6 +143,7 @@ void ShaderOptimizer::ApplyProfileToShaderCreateInfo(
                 }
 #endif
 
+#if LLPC_BUILD_GFX10
```

Comment:
We should not use LLPC macro in api, please VKI_BUILD_GFX10, I missed it in previous review:(.